cypress/integration/programDimensions.cy.js

```diff
@@ -15,6 +15,44 @@ import {
 import { goToStartPage } from '../helpers/startScreen.js'
 import { EXTENDED_TIMEOUT } from '../support/util.js'
 
+/*
+Test data used:
+    E2E program
+        Stages: 1 (i.e. stage is auto-selected)
+        Scheduled date: enabled (for selected stage)
+        Incident date: enabled
+
+    WHO RMNCH Tracker
+        Stages: >1
+        Scheduled date: enabled (for selected stage)
+        Incident date: disabled
+
+Note that scheduled date can be toggled per program stage, but thus requires a stage to be selected before being enabled.
+I.e. Schedule date works like this:
+    no stage: disabled
+    stage without scheduled date: disabled
+    stage with scheduled date: enabled
+*/
+
+const TEST_PROGRAM = {
+    programName: 'WHO RMNCH Tracker',
+    noStage: {
+        [DIMENSION_ID_EVENT_DATE]: 'Event date',
+        [DIMENSION_ID_ENROLLMENT_DATE]: 'Date of first visit',
+        [DIMENSION_ID_SCHEDULED_DATE]: 'Scheduled date',
+        [DIMENSION_ID_INCIDENT_DATE]: 'Date of incident',
+        [DIMENSION_ID_LAST_UPDATED]: 'Last updated on',
+    },
+    stage: {
+        stageName: 'First antenatal care visit',
+        [DIMENSION_ID_EVENT_DATE]: 'Date of visit',
+        [DIMENSION_ID_ENROLLMENT_DATE]: 'Date of first visit',
+        [DIMENSION_ID_SCHEDULED_DATE]: 'Appointment date',
+        [DIMENSION_ID_INCIDENT_DATE]: 'Date of incident',
+        [DIMENSION_ID_LAST_UPDATED]: 'Last updated on',
+    },
+}
+
 const assertDimensionsForEventWithoutProgramSelected = (
     scheduleDateIsSupported
```

Comment:
Minor: Rename to `scheduledDate...` (note the d) across the file.

Reply:
fixed